API.md

```diff
@@ -0,0 +1,54 @@
+# AmperkaCCS811 API
+
+## `class CCS811`
+
+Create an object of type `CCS811` to communicate with a particular [Air Quality Sensor](https://amperka.ru/product/sensor-co2-ccs811-with-case).
+
+### `CCS811(uint8_t i2cAddress = 0x5A)`
+
+Constructs a new object. The argument `i2cAddress` is the specifies the board I²C address in HEX value, which is `0x5A` factory-default but can be changed is `0x5B`. If omitted, the board I²C address is `0x5A`.
+
+### `void begin(TwoWire* wire = &Wire)`
+
+Initializes the given interface, prepares the board for communication. The argument `wire` is an index of the TwoWire interface.
+If omitted, the connection started on default hardware I²C.
+
+### `void setDriveMode(uint8_t mode)`
+
+Sets the drive mode, which is responsible for the sample rate and power consumption of the sensor. The argument `mode` is the range from `0` to `3`:
```

Comment:
CCS811 has 5 modes according to datasheet.

Reply:
I am add 5 as enum name